sdk/core/logger/package.json

```diff
@@ -0,0 +1,116 @@
+{
+  "name": "@azure/logger",
+  "sdk-type": "client",
+  "version": "1.0.0-preview.1",
+  "description": "Microsoft Azure SDK for JavaScript - Aborter",
+  "main": "./dist/index.js",
+  "module": "./dist-esm/src/logger.js",
+  "browser": {
+    "./dist/index.js": "./browser/index.js",
```

Comment:
Didnt we learn from another PR that this should be `./browser/logger.js`?

Reply:
Apparently I've learned nothing. Will fix.